src/main/cluster.ts

```diff
@@ -252,16 +252,12 @@ export class Cluster implements ClusterModel, ClusterState {
   }
 
   protected async k8sRequest<T = any>(path: string, options: RequestPromiseOptions = {}): Promise<T> {
+    (options.headers ??= {}).Host = `${this.id}.${new URL(this.kubeProxyUrl).host}` // required in ClusterManager.getClusterForRequest()
+    options.json ??= true
```

Comment:
This is nullish assignment? seems to very new. Good to know

Reply:
Yup, a couple of months old only